src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/SARIF1009.IndexPropertiesMustBeConsistentWithArrays_Invalid.sarif

```diff
@@ -69,35 +69,6 @@
         }
       ],
       "results": [
-        {
-          "ruleId": "SARIF1009",
```

Comment:
>SARIF1009 [](start = 21, length = 9)

Why did this result disappear?

Reply:
i rollback my changes and added to ignore the rule 2004 as well.

---
In reply to: [455262015](https://github.com/microsoft/sarif-sdk/pull/1995#discussion_r455262015) [](ancestors = 455262015)